help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -99,12 +100,13 @@ Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL
 ### Authenticate to destination {#authenticate}
 
 To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
-![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination.png)
+![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination1.png)
 
 Fill in the required fields below. Refer to the [Gather Dynamics 365 credentials](#gather-credentials) section for any guidance.
 * **[!UICONTROL Client ID]**: The [!DNL Dynamics 365] Client ID for your [!DNL Azure Active Directory] application.
 * **[!UICONTROL Tenant ID]**: The [!DNL Dynamics 365] Tenant ID for your [!DNL Azure Active Directory] application. 
 * **[!UICONTROL Client Secret]**: The [!DNL Dynamics 365] Client Secret for your [!DNL Azure Active Directory] application.
+* **[!UICONTROL Region]**: Your [!DNL Dynamics 365] Region.
```

Comment:
based on the Microsoft Dynamics documentation, please provide a brief example, e.g. for companies provisioned in North America, select crm and for companies provisioned in Canada, they would select crm3

https://learn.microsoft.com/en-us/power-platform/admin/new-datacenter-regions

Reply:
Yes updated with excepted changes